src/AppInstallerRepositoryCore/Manifest/YamlManifestParser.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "ManifestValidation.h"
+#include "Manifest.h"
+
+namespace AppInstaller::Manifest
+{
+    static const uint64_t MaxSupportedMajorVersion = 1;
+    static const ManifestVer PreviewManifestVersion = ManifestVer("0.1.0", false);
+    static const ManifestVer PreviewManifestVersionV2 = ManifestVer("0.2.0", false);
+
+    struct YamlManifestParser

```

Comment:
>YamlManifestParser [](start = 11, length = 18)

Can we drop Manifest from so many names since we are in the Manifest namespace?

Manifest::YamlParser::Create
Manifest::YamlParser::CreateFromPath

You don't need to have names repeated if you don't using namespace the previous names out.

Reply:
Sure

---
In reply to: [454024340](https://github.com/microsoft/winget-cli/pull/493#discussion_r454024340) [](ancestors = 454024340)